<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lista">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>111</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Guardar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Tel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Email">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Zeit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Kita">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Adress">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zeit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Nuevo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Neu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cargar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Laden</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
